ui/admin/app/components/agent/ToolForm.tsx

```diff
@@ -159,6 +175,9 @@ export function ToolForm({
                             })
                         }
                         onRemoveTools={removeTools}
+                        onUpdateTools={(tools) =>
```

Comment:
nit: Silly question, if onUpdateTools handles both addition and removal of tools, could onAddTool and onRemoveTools be axed?

Reply:
updated